src/ansys/fluent/core/session_solver_icing.py

```diff
@@ -0,0 +1,36 @@
+"""Module containing class encapsulating Fluent connection.
+
+**********PRESENTLY SAME AS SOLVER WITH A SWITCH TO SOLVER***********
+"""
+from ansys.fluent.core.fluent_connection import _FluentConnection
+from ansys.fluent.core.session_solver import Solver
+
+
+class SolverIcing(Solver):
+    """Encapsulates a Fluent server for Icing session connection.
+
+    SolverIcing(Session) holds the top-level objects for solver TUI,
+    settings and icing datamodel objects calls.
+    """
+
+    def __init__(
+        self,
+        fluent_connection: _FluentConnection,
+    ):
+        super(SolverIcing, self).__init__(fluent_connection=fluent_connection)
+        self._flserver_root = None
+
+    @property
+    def _flserver(self):
+        """root settings object."""
```

Comment:
I guess this is not accurate

Reply:
fixed the comment, its the datamodel root.

FYI icing is using both the datamodel (for all its icing settings), and the settings api for regular fluent settings access , report definitions, etc.